tests/yast2_cmd/yast_lan.pm

```diff
@@ -0,0 +1,30 @@
+ # SUSE's openQA tests
+ #
+ # Copyright © 2009-2013 Bernhard M. Wiedemann
+ # Copyright © 2012-2019 SUSE LLC
```

Comment:
not needed coputight from 2009 to 2019 in new test 
simple 2019 SUSE LLC is sufficient

Reply:
Fixed